### DISCUSSION TITLE
Hello everyone, /api/v2/users, where is the implementation of this interface definition

### DISCUSSION THREAD

**Discussion by Pythoneryeah** [coder/coder#8298] [General] [answered]
Hello everyone, /api/v2/users, where is the implementation of this interface definition

**Comment by matifali** [comment]
what about searching https://github.com/search?q=repo%3Acoder%2Fcoder+api%2Fv2%2Fusers

**Reply by Pythoneryeah** [reply]
Thank you for your answer, but I have searched, but there is really no interface implementation code logic. Is the interface running in an official backend service?

**Reply by matifali** [reply, marked as answer]
Check here
https://github.com/coder/coder/blob/647fd7699b4d480e61c6e536a96dcd65390b2f1e/coderd/users.go
